recipe/meta.yaml

```diff
@@ -302,19 +273,59 @@ outputs:
       requires:
         - {{ compiler('c') }}
       commands:
-        - VER=${PKG_VERSION%.*}   # [not win]
+        - VER=${PKG_VERSION%.*}    # [not win]
+        - VERABI=${VER}            # [not win]
+{% if freethreading == "yes" %}
+        - VERABI=${VERABI}t        # [not win]
+{% endif %}
+        - VERABI_NO_DBG=${VERABI}  # [not win]
 {% if py_interp_debug == "yes" %}
-        - VERABI=${VER}d    # [not win]
-{% else %}
-        - VERABI=${VER}   # [not win]
+        - VERABI=${VERABI}d        # [not win]
 {% endif %}
         - test -f ${PREFIX}/lib/libpython${VERABI}.a  # [unix]
-        - test -f ${PREFIX}/lib/libpython${VERABI}.nolto.a  # [unix]
-        - test -f ${PREFIX}/lib/python${VER}/config-${VERABI}-darwin/libpython${VERABI}.a  # [osx]
+        - test -f ${PREFIX}/lib/libpython${VERABI_NO_DBG}.nolto.a  # [unix]
+        - test -f ${PREFIX}/lib/python${VERABI_NO_DBG}/config-${VERABI}-darwin/libpython${VERABI}.a  # [osx]
         - pushd tests/prefix-replacement    # [unix]
         - bash build-and-test.sh      # [unix]
         - popd    # [unix]
 
+  - name: cpython
+    build:
+      noarch: generic
+    requirements:
+      - python {{ version }}.*
+      - python_abi * *_{{ abi_tag }}
+
+  - name: cpython-freethreading
```

Comment:
How about just `freethreading` or `free-threading`? The install command would then be `conda install python freethreading`

Reply:
I'd prefer keeping the `cpython-`

Reply:
cc @conda-forge/core

Reply:
@jjhelmus what do you think? I'm partial to `cpython-freethreading`, but not fixed on it.

Reply:
I feel like `cpython-freethreading` is a good metapackage for experts who understand the difference between cpython and python. I'm not sure in beginners would discover this package.  Personally, I think something like `freethreading` or `python-freethreading` would be more obvious to user. That said these could be added in addition to `cpython-freethreading` not in place of the meta-package.